src/templates/BlogPost/BlogPost.js

```diff
@@ -23,14 +23,13 @@ const BlogPost = ({ pageContext }) => {
   if (previous) thumbnailsItems.push(previous);
 
   const pageWrapperClass = classNames(styles.container, 'pageWrapper');
+  const excr = data.excerpt.substr(3, data.excerpt.length - 8);
```

Comment:
please use regex https://css-tricks.com/snippets/javascript/strip-html-tags-in-javascript/